user/homework.go

```diff
@@ -57,6 +59,36 @@ func (hw *Homework) Save() error {
 	return nil
 }
 
+// SaveAfterFetch filters out problems for tracks the user has yet to start
+// before saving them all to disk.
+func (hw *Homework) SaveAfterFetch(numArgs int) error {
+	hw.RejectUnstartedTracks(numArgs)
+	if len(hw.Items) == 0 {
+		fmt.Println(`
+You have yet to start a language track!
+View all available language tracks with "exercism tracks"
+Fetch exercises for your first track with "exercism fetch LANGUAGE"`)
+	}
+	return hw.Save()
+}
+
+// RejectUnstartedTracks removes any items that are part of tracks the user
+// has yet to start and doesn't have a folder for on their local machine. This
+// only happens when a user calls `exercism fetch` without any arguments.
+func (hw *Homework) RejectUnstartedTracks(numArgs int) {
```

Comment:
Good call on using a boolean instead. How about `isCalledWithoutArgs` or `isFetchedWithoutArgs`? Both are kind of long, but I can't think of anything shorter to describe the behavior we're flagging here.